libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.tsx

```diff
@@ -0,0 +1,186 @@
+                  <CopyToClipboard
+                    className="ml-2 text-text-300 invisible group-hover:visible"
+                    content={
+                      (props.defaultAdvancedSettings &&
+                        props.defaultAdvancedSettings[key as keyof ApplicationAdvancedSettings]?.toString()) ||
+                      ''
+                    }
+                  />
+                </>
+              ),
+              className: 'group',
+            },
+            {
+              className: '!p-1',
+              content: (
+                <Controller
+                  name={key}
+                  control={control}
+                  rules={{
+                    required: 'Please enter a value.',
+                  }}
+                  defaultValue=""
+                  render={({ field, fieldState: { error } }) => (
+                    <InputTextSmall
+                      className="shrink-0 grow flex-1"
+                      data-testid="value"
+                      name={field.name}
+                      onChange={field.onChange}
+                      value={field.value}
+                      error={error?.message}
+                      errorMessagePosition="left"
+                      transparentBorderOnValid={true}
+                      label={field.name}
+                    />
+                  )}
+                />
+              ),
+            },
+          ],
+        }
+      })
+    : []
+
+  table = [...table, ...tableBody]
+
+  return (
+    <div className="flex flex-col justify-between w-full">
+      <div className="p-8 ">
+        <div className="flex justify-between mb-4">
+          <div>
+            <h1 className="h5 text-text-700 mb-2">Advanced Settings</h1>
+            <p className="text-sm text-text-500 max-w-content-with-navigation-left">
+              Settings are injected at the build and run time of your application and thus any change on this section
+              will be applied on the next manual/automatic deploy.
+            </p>
+          </div>
+        </div>
+
+        <InputToggle
+          className="mb-4"
+          dataTestId="show-overriden-only-toggle"
+          value={showOverriddenOnly}
+          onChange={toggleShowOverriddenOnly}
+          title="Show only overridden settings"
+        />
+
+        <form onSubmit={props.onSubmit}>
+          <div className="relative">
+            {(!props.loading || props.loading === 'loading') && props.keys?.length === 0 ? (
+              <div className="flex justify-center">
+                <LoaderSpinner className="w-6"></LoaderSpinner>
+              </div>
+            ) : (
+              <EditionTable tableBody={table} />
+            )}
+            <div
+              className={`sticky bottom-10 mt-10 flex justify-center ${formState.isDirty ? 'visible' : 'hidden'}`}
+              data-testid={'sticky-action-form-toaster'}
+            >
+              <StickyActionFormToaster
```

Comment:
I think for this component we missing a loading status for the button during the action sending